server/api/users.js

```diff
@@ -92,4 +92,31 @@ router.post('/:id/cart', async (req, res, next) => {
   }
 })
 
-
+router.delete('/:id/cart', async (req, res, next) => {
+  try {
+    const lineItem = await LineItem.findOne({
+      where: {
+        orderId: +req.body.orderId,
+        productId: +req.body.productId
+      }
+    })
+    if (!lineItem) {
+      const err = new Error(`Unable to remove the item with id ${req.params.id} from the order.`)
+      err.status = 400
+      return next(err)
+    }
+    if (lineItem.quantity > 1) {
```

Comment:
Decrementing the quantity (and incrementing it) should be in a put request -- the delete functionality should completely delete the line item